glfw/src/Window.zig

```diff
@@ -390,6 +448,8 @@ pub inline fn create(width: usize, height: usize, title: [*c]const u8, monitor:
         if (share) |w| w.handle else null,
     );
     try getError();
+    
+    try Window.defaultHints();
```

Comment:
Ah, I see your intent now is to "cleanup" the hints so-to-speak by changing them back to the default. This makes sense.

However, if window creation fails we would not clean up. Maybe instead `defer` and ignore the error? I think `defer _ = Window.defaultHints();` would do the trick (but I might have the exact syntax wrong)

Reply:
In my original thinking, a `defer` wouldn't have worked, as you can't `try` in a `defer` statement; but looking at it closer, the only possible error for `Window.defaultHints` is `Error.NotInitialized`, which should already have been caught in `Hints.set`, so an error should never be thrown there, a la `catch unreachable`.